src/gui/src/SettingsDialog.h

```diff
@@ -38,19 +37,19 @@ class SettingsDialog : public QDialog
         SettingsDialog(QWidget* parent, AppConfig& config);
         ~SettingsDialog() override;
 
+    signals:
```

Comment:
Use Q_SIGNALS in new code.

Reply:
They both do the same thing and any time you generate any code in Qt it will use a `signals`  section and not a `Q_SIGNALS` section , so unless you have a good reason I'd rather not waste my time typing two extra characters.

Reply:
We will enable QT_NO_KEYWORDS at some point.

Reply:
Done